ibtk/src/utilities/IBTK_MPI.cpp

```diff
@@ -43,30 +44,33 @@ IBTK_MPI::getSAMRAIWorld()
 #if SAMRAI_VERSION_MAJOR == 2
     return SAMRAI::tbox::SAMRAI_MPI::commWorld;
 #else
-    return SAMRAI::tbox::SAMRAI_MPI::getSAMRAIWorld()
+    return SAMRAI::tbox::SAMRAI_MPI::getSAMRAIWorld();
```

Comment:
Does this mean you are messing around with SAMRAI 3/4??? 😀

Reply:
No, not yet; it's just that what we have clearly won't work without the `;`.